test_napi/src/properties.rs

```diff
@@ -36,45 +53,56 @@ pub fn init(env: napi_env, exports: napi_value) {
 
   let properties = &[
     napi_property_descriptor {
-      utf8name: "test_property_rw\0".as_ptr() as *const c_char,
+      utf8name: "test_simple_property\0".as_ptr() as *const c_char,
       name: ptr::null_mut(),
       method: None,
       getter: None,
       setter: None,
       data: ptr::null_mut(),
       attributes: enumerable | writable,
-      value: number,
+      value: init_constants(env),
     },
     napi_property_descriptor {
-      utf8name: "test_property_r\0".as_ptr() as *const c_char,
+      utf8name: "test_property_rw\0".as_ptr() as *const c_char,
```

Comment:
You can use `cstr!()` helper here

Reply:
I think it's overkill to use std's `CString`. `as_ptr` is obvious for a known comptime string. `cstr!` also allocates for the last NULL byte

Reply:
This is testing code, I believe readability here is more important than allocating one allocation.

Reply:
ok ill do it but this makes no difference; just that now ive to know about this new macro.

```rs
cstr!("test_property_rw")
// vs
"test_property_rw\0".as_ptr()
```

Reply:
You don't have to do `as *const c_char` cast as well 😋